frontend/packages/knative-plugin/src/components/overview/__tests__/overview-knative-test-data.ts

```diff
@@ -0,0 +1,49 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
this won't be needed, have refactored it in  https://github.com/openshift/console/pull/3752, Just FYI

Reply:
I added a test data to this file in a PR for separate component. So I refactored it there. I could remove the file from this PR  as well

Reply:
you can even just undo changes in Routes* components, will be handled in https://github.com/openshift/console/pull/3752